package/com.unity.formats.usd/Runtime/Scripts/Behaviors/UsdAsset.cs

```diff
@@ -30,9 +31,15 @@ public class UsdAsset : MonoBehaviour {
     public double Length { get { return ComputeLength(); } }
 
     [Header("Source Asset")]
-    public string m_usdFile;
-    public string m_projectAssetPath = "Assets/";
+    [SerializeField]
+    string m_usdFile;
 
+    public string fullPath
```

Comment:
I feel like "fullPath" is kind of ambiguous -- I think we should either add a doc comment explaining that it's the full path to the USD file or rename to usdFileFullPath? What do you think?

Reply:
renamed to usdFullPath